### ISSUE TITLE
camltkwrap.cmx/cmo/cmi not installed

### ISSUE THREAD

**Issue by PascalRaymond** [garrigue/labltk#14]
Hi,
Implemetation of camltkwrap is not installed since support/Makefile filters only the cmx/i/o
that correspond to public mli, e.g. for .cmx the rule is:

~~~
ocamlfind install labltk -add $(PUBMLI:.mli=.cmx) -optional tkthread.cmx tkthread.$(O)
~~~

It makes it impossible to use Camltk features that are just aliases to Camltkwrap.
In particular Camltk.Textvariable:

~~~
# Camltk.Textvariable.create();;
Error: The module Camltk.Textvariable is an alias for module Camltkwrap.Textvariable, which is missin
~~~

Copying camltkwrap.cmx/cmo/cmi to install dir solved the problem

regards

Pascal

**Comment by michipili** [comment]
Hi @PascalRaymond thanks for reaching out! 

Could you add a few details: what operating system are you using and how did you try to install labltk?

(As side note I reformatted your issue to make it easier to read. You can use the “Preview” tab to check the formatting of your text and visit https://guides.github.com/features/mastering-markdown/ to see how the formatting works. Interestingly there is no easy way to just deactivate all styling on an issue.)

**Comment by garrigue** [comment]
Thank you for your report.
This is fixed in commit d587aeb .
Note that there is no need to install `camltkwrap.cmo`, as it is included in `labltk.cma`.
What is confusing is that not all the information contained in `camltkwrap.cmx` is included in `labltk.cmxa`.

**Comment by garrigue** [comment]
The fix went into 8.06.10
